pkg/oc/admin/prune/images.go

```diff
@@ -625,3 +648,35 @@ func getRegistryClient(clientConfig *restclient.Config, registryCABundle string,
 		Transport: wrappedTransport,
 	}, nil
 }
+
+func parsePatternsFile(filename string) ([]*regexp.Regexp, error) {
+	file, err := os.Open(filename)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	var patterns []*regexp.Regexp
+
+	scanner := bufio.NewScanner(file)
```

Comment:
Don't use bufio.NewScanner for reading lines. It limits the length of tokens (iow. lines).
Use `file.ReadString('\n')` instead.